### PR TITLE
Improve protocol presentation with roles of client and server

### LINKED ISSUE #142: Don't put critical information in a diagram
The [diagram in Section 4](https://www.ietf.org/archive/id/draft-irtf-cfrg-rsa-blind-signatures-06.html#section-4-5) contains information that is critical to understanding the process.

I think that all you need to do here is change the definitions of Blind, BlindSign, and Finalize to include who performs them.

### LINKED ISSUE #143: Show the outputs of functions
Blind is currently defined:

```
Blind(pkS, msg)
```

It might be easier to understand as:

```
blinded_msg, inv = Blind(pkS, msg)
```

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #142
Closes #143 

I turned the list of function descriptions (which was somewhat redundant with the text in each of the following subsections in the doc) into a linear description of the protocol, explaining who does what. I then summarized the description with the existing picture. I think this addresses both of @martinthomson's comments and makes the key information more clear.

### CODE REVIEW

File: draft-irtf-cfrg-rsa-blind-signatures.md [hunk 1]
```diff
@@ -184,18 +184,34 @@ where they interact to compute `sig = Sign(skS, msg)`, where `msg` is the privat
...
+~~~
+
+The server then sends `blind_sig` to the client, which the finalizes the protocol by computing:
+
+~~~
+sig = Finalize(pkS, msg, blind_sig, inv)
+~~~
 
-Using these three functions, the core protocol runs as follows:
+Upon completion, correctness requires that clients can verify signature `sig` over private
```
Comment (jedisct1) [inline draft-irtf-cfrg-rsa-blind-signatures.md:209]: The protocol description is indeed way easier to grasp that way..

The part about "correctness" remains confusing, though. Is it an optional step? What are the implications not to do it?

Why not write that clients MUST verify the signature. After all, RSA signature verification is fast.

Comment (chris-wood) [reply draft-irtf-cfrg-rsa-blind-signatures.md:209]: I don't see the confusion, so would you mind proposing text?

**Review by martinthomson — COMMENTED** [review]
This works for me.  I'll let you both work it out.

**Comment by chris-wood** [comment]
@FredericJacobs can you please review? I'm going to leave the correctness text as-is, since it speaks about correctness with respect to the output of the function.

**Review by FredericJacobs — APPROVED** [review]

**Pull request merged**